tests/wpunit/MediaItemQueriesTest.php

```diff
@@ -336,4 +336,37 @@ public function testMediaItemQuery( $image_meta = [] ) {
 
 	}
 
+    /**
+     * testPostQuery
+     *
+     * This tests creates a single attachment and retrieves said post URL via a GraphQL query
+     *
+     * @since 0.3.6
+     */
+    public function testMediaItemUrl() {
```

Comment:
This is good! 

We should include another test to test something of another mime type, like a video or mp3 or something. 

I think the common complaint has been that `sourceUrl` wasn't working for non-images